toolchain/semantics/semantics_handle_operator.cpp

```diff
@@ -80,6 +100,17 @@ auto SemanticsHandlePrefixOperator(SemanticsContext& context,
               value_id));
       break;
 
+    case TokenKind::Const: {
+      auto inner_type_id = context.ExpressionAsType(parse_node, value_id);
+      context.AddNodeAndPush(
+          parse_node,
+          SemanticsNode::ConstType::Make(
+              parse_node,
+              context.CanonicalizeType(SemanticsNodeId::BuiltinTypeType),
```

Comment:
Can this SemanticsTypeId::TypeType? That is, I think BuiltinTypeType doesn't need canonicalization.
```suggestion
              SemanticsTypeId::TypeType,
```